docs/root/api-v3/bootstrap/bootstrap.rst

```diff
@@ -11,3 +11,4 @@ Bootstrap
   ../config/overload/v3/overload.proto
   ../config/ratelimit/v3/rls.proto
   ../extensions/wasm/v3/wasm.proto
+  ../extensions/vcl/v3alpha/vcl_socket_interface.proto
```

Comment:
i think this should also be alphabetical order

Reply:
Fixed